automation_tools/satellite6/upgrade/tests/utils.py

```diff
@@ -0,0 +1,152 @@
+import os
```

Comment:
Better to add a module docstring with more information about what type of utils we would find/place on this module.